plaso/parsers/aws_elb_access.py

```diff
@@ -0,0 +1,312 @@
+# -*- coding: utf-8 -*-
+"""Parser for AWS ELB access logs.
+
+This parser is based on the log format documented at
+https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html
+"""
+import pyparsing
+
+from dfdatetime import time_elements as dfdatetime_time_elements
+
+from plaso.containers import events
+from plaso.containers import time_events
+from plaso.lib import definitions
+from plaso.lib import errors
+from plaso.parsers import manager
+from plaso.parsers import text_parser
+
+
+class AWSELBEventData(events.EventData):
+  """AWS Elastic Load Balancer access log event data
+
+  Attributes:
+    type (str): The type of request or connection.
+    elb (str): The resource ID of the load balancer.
+    client_ip_address (str): The IP address of the requesting client.
+    client_port (int): The port of the requesting client.
+    target_ip_address (str): The IP address of the target that processed
+        this request.
+    target_port (int): The port of the target that processed this request.
+    request_processing_time (str): The total time elapsed from
```

Comment:
are these times or durations ?

Reply:
These are durations (i.e. time elapsed), I edited the docstring to reflect that.